src/components/NavigationItem/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface TNavigationLinkPropTypes {
+  As: () => JSX.Element
+  type: 'main' | 'sub' | 'block-header' | 'user' | 'product' | 'action'
```

Comment:
I think it better to have an enum for this case because I saw that you are using for example 'user' or other strings in different files and if in some cases you will need to change one of them you should change in all files and in all use cases

Reply:
done